L1Trigger/TrackerTFP/python/KalmanFilterFormats_cfi.py

```diff
@@ -36,18 +36,18 @@
     ),
 
```

Comment:
There should be a comment at the start of this file explaining what it is for, and giving a clue what things like BaseShiftxyz represent.

Reply:
done

Reply:
I still see no info about how these numbers are obtained.

Reply:
done.